### ISSUE TITLE
stochastic gradient descent and Loss Function

### ISSUE THREAD

**Issue by Yurinodac** [Yurinodac/FIN550#1]

**Comment by Yurinodac** [comment]
[script.py](https://github.com/user-attachments/files/22584838/script.py)
